### ISSUE TITLE
New version broke texture preview/extraction

### ISSUE THREAD

**Issue by MilutinMujovic** [Perfare/AssetStudio#585]
Hello!
I downloaded the new version and it completely broke texture preview and extraction.
When I select the texture I get this message:
![image](https://user-images.githubusercontent.com/38858100/89715663-8f969e00-d9a7-11ea-85d1-8ca3203e1e98.png)

I never had issues with AssetStudio before, can you please fix this? :)

**Comment by hozuki** [comment]
This should be caused by my PR (#562).

On my PC it runs fine (both 32-bit and 64-bit).
Could you describe your environment? Did you install [Visual C++ 2017 (or 2015-2019) Redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) (search for "visual c++" in `Settings` app - `Applications and Features`)?

**Comment by MilutinMujovic** [comment]
I believe I had them installed.
![image](https://user-images.githubusercontent.com/38858100/89743608-12515300-daa5-11ea-8c5f-a93c60c74c58.png)

**Comment by hozuki** [comment]
TL;DR:
According to #563 the problem can be fixed by re-installing the VC++ 2015-2019 redist package (link above, and in readme). https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

---

I examined the dependencies of the new native DLLs:

```
Microsoft (R) COFF/PE Dumper Version 14.27.29110.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file .\Texture2DDecoderNative.dll

File Type: DLL

  Image has the following dependencies:

    VCRUNTIME140_1.dll
    VCRUNTIME140.dll
    api-ms-win-crt-math-l1-1-0.dll
    api-ms-win-crt-heap-l1-1-0.dll
    api-ms-win-crt-stdio-l1-1-0.dll
    api-ms-win-crt-runtime-l1-1-0.dll
    KERNEL32.dll

  Summary

        1000 .data
        1000 .pdata
        5000 .rdata
        1000 .reloc
        1000 .rsrc
       1A000 .text
```

Old C++/CLI DLL:

```plain
Microsoft (R) COFF/PE Dumper Version 14.27.29110.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file .\Texture2DDecoder.dll

File Type: DLL

  Image has the following dependencies:

    KERNEL32.dll
    VCRUNTIME140.dll
    api-ms-win-crt-stdio-l1-1-0.dll
    api-ms-win-crt-runtime-l1-1-0.dll
    api-ms-win-crt-heap-l1-1-0.dll
    MSVCP140.dll
    api-ms-win-crt-math-l1-1-0.dll
    mscoree.dll

  Summary

        5000 .data
        3000 .nep
        1000 .pdata
       29000 .rdata
        1000 .reloc
        1000 .rsrc
       17000 .text
```

The critical difference is `VCRUNTIME140_1.dll`. I checked the redist DLLs (in `[VSDir]\VC\Redist\MSVC\14.16.27012`, [here](https://docs.microsoft.com/en-us/visualstudio/productinfo/2017-redistribution-vs)), and there is no `vcruntime140_1.dll`. That means, **old VC++ 2017 redist does not ship with `vcruntime140_1.dll`**, because binaries compiled by VC++ 2017 toolchain in VS 2017 use the same base runtime as in VC++ 2015. (Fun fact: VC++ 2015-2019 share the same major MSVC version, 14.)
A patch 1 DLL (`vcruntime140_1.dll`) is used by binaries compiled by VC++ 2019. However, something changed in VS 2019 so if you use VC++ 2017 toolchain in VS 2019, compiled binaries will depend on `vcruntime140_1.dll`.
Therefore, you have to install the new runtime DLL, which is included in VC++ 2019 redist, which is now merged into VC++ 2015-2019 redist. That's why reinstalling the latest VC++ 2015-2019 redist fixes this problem.
**Installing the old VC++ 2017 redist only (from whatever archive you found it) breaks programs compiled using VC++ 2019 with VC++ 2017 toolchain!**

OMFG this dependency behavior is so damn confusing! Please, Microsoft, fix this problem. Oh, they probably won't.

**Comment by Perfare** [comment]
I modify the `Runtime Library` property, set it to /MT, no longer need to install vc redist now, so this problem should have been fixed.
